dataclients/kubernetes/kube.go

```diff
@@ -215,6 +215,9 @@ type Options struct {
 	// ForceKubernetesService overrides the default Skipper functionality to route traffic using
 	// Kubernetes Endpoint, instead using Kubernetes Services.
 	ForceKubernetesService bool
+
+	// BackendTrafficAlgorithm specifies the algorithm to calculate the backend traffic.
+	BackendTrafficAlgorithm string
```

Comment:
The plan is to make this configurable via flag.

Reply:
What are potential values for this flag? Random, Round Robin, etc?

Reply:
I named the new algorithm `"traffic-segment-predicate"` and the current default would be `"traffic-predicate"`